AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/SubscriptionSync/ReconcileAndLocalSave/ReconcileAndLocalSaveOperation.swift

```diff
@@ -278,7 +297,7 @@ class ReconcileAndLocalSaveOperation: AsynchronousOperation {
         let version = savedModel.syncMetadata.version
         if savedModel.syncMetadata.deleted {
             mutationType = .delete
-        } else if version == 1 {
+        } else if version == 1 || existsLocally {
```

Comment:
I think you have an error: shouldn't this be 
```suggestion
        } else if version == 1 || !existsLocally {
```

?

Reply:
fixed